classes/simplemap.php

```diff
@@ -2587,7 +2587,7 @@ function get_default_shortcode_atts() {
 		 * @return string|void
 		 */
 		function backwards_compat_categories_text( $text ) {
-			return __( 'Categories', 'simplemap' );
+			return __( 'Categories', 'SimpleMap' );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->